### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/tutorial`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4569]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4569|
|Type|enhancement|
|Created|2010-07-11 07:00:34Z|
|Branch|https://github.com/twisted/twisted/tree/tutorial-lore2sphinx-4569|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for `projects/core/howto/tutorial`

Attachments:

* [tutorial.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/232/232758d2b2310c93c7a3fb207558f22b331793e4/587458cb184e6a561ecd09d47cefafbaf1fb1dd4.patch) (6847 bytes) - added by jdb on 2010-09-09 13:21:12Z - 
* [tutorial2.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/232/232758d2b2310c93c7a3fb207558f22b331793e4/7d71fa6b963b62087f12687a440dc40716ca4f4b.patch) (18571 bytes) - added by jdb on 2010-09-27 11:14:20Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4569 4569
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_tutorial_lore2sphinx_4569 branches/tutorial-lore2sphinx-4569
branch_author__jdb jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1278831634000000 1278831634000000
changetime__1285984376000000 1285984376000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jdb's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|jdb set status to `closed`|
|-|-|

Modified the xhtml input (made sure no <code> or <em> tag begins a newline), to correct the rst output.  In the directory core/howto/tutorial/, the files intro.xhtml, protocol.xhtml and web.xhtml were modified. Now, there is no more missing space between a word and either `` or **.

lore2sphinx does not copy the python code listings from the lore input directory to the rst output directory. When the listings are copied by hand in the rst directory, the listings appear correctly in the final html pages.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `reopened`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner<br>@exarkun set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to jdb|
|-|-|

Unfortunately the patch file appears to be corrupted.

```
exarkun@boson:~/Projects/Twisted/trunk$ patch -p1 < tutorial.xhtml.patch 
patch: **** Only garbage was found in the patch input.
```

**Comment by twisted-trac** [comment]
|<img alt="jdb's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|jdb removed owner|
|-|-|

The latest patch applies correctly to the latest trunk:
```
$ svn checkout svn://svn.twistedmatrix.com/svn/Twisted/trunk
$ cd trunk
$ svn status
?       tutorial2.xhtml.patch

$ patch -p0 < tutorial.xhtml.patch 
patching file doc/core/howto/tutorial/intro.xhtml
patching file doc/core/howto/tutorial/protocol.xhtml
patching file doc/core/howto/tutorial/web.xhtml

$ svn status
?       tutorial.xhtml.patch
M       doc/core/howto/tutorial/intro.xhtml
M       doc/core/howto/tutorial/web.xhtml
M       doc/core/howto/tutorial/protocol.xhtml
```

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph commented|
|-|-|

(In [30076]) Branching to 'tutorial-lore2sphinx-4569'

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_8"></a>|@glyph commented|
|-|-|

[Generated documentation should be here](http://buildbot.twistedmatrix.com/builds/sphinx-html-15670/), when it's ready.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun commented|
|-|-|

(In [30078]) Undo a typo introduced

refs [#4569](https://github.com/twisted/twisted/issues/4569)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun commented|
|-|-|

(In [30079]) Fix another case where the conversion gets confused

refs [#4569](https://github.com/twisted/twisted/issues/4569)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun commented|
|-|-|

(In [30080]) Same again

refs [#4569](https://github.com/twisted/twisted/issues/4569)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_12"></a>|@exarkun set owner to @exarkun|
|-|-|

With the above extra fixes, looks good.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_13"></a>|@exarkun set status to `closed`|
|-|-|

(In [30082]) Merge tutorial-lore2sphinx-4569

Author: jdb
Reviewer: exarkun
Fixes: [#4569](https://github.com/twisted/twisted/issues/4569)

Tweaks to the Lore markup in the core tutorial to make it more amenable to conversion to Sphinx.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
